harmonic/install_windows_src.md

```diff
@@ -132,6 +142,34 @@ call install\setup.bat
 This is the end of the source install instructions; head back to the [Getting started](/docs/all/getstarted)
 page to start using Gazebo!
 
+> [!WARNING]
+> As Gazebo GUI is not yet working, running `gz sim` will not work. You can run only the server with
+
+```cmd
+gz sim -s -v
+```
```

Comment:
Same as above.
I tried the `<div>` CSS here to get a nice yellow warning bar, but once we use `<div>`, we don't get the Markdown parsing that makes the code font face. So it looks better with just simple Markdown.
Code block needs to be indented too. I've tested this on the rendered page.
```suggestion
> **WARNING**
> As Gazebo GUI is not yet working, running `gz sim` will not work. You can run only the server with
> ```cmd
> gz sim -s -v
> ```
```